src/js/angular/guides/steps/complex/ttyg/select-ttyg-agent/plugin.js

```diff
@@ -0,0 +1,62 @@
+PluginRegistry.add('guide.step', [
+    {
+        guideBlockName: 'select-ttyg-agent',
+        getSteps: (options, services) => {
+            const GuideUtils = services.GuideUtils;
+            options.mainAction = 'select-ttyg-agent';
+
+            return [
+                {
+                    guideBlockName: 'info-message',
+                    options: angular.extend({}, {
+                        content: 'guide.step_plugin.select-ttyg-agent.info.content',
+                        url: '/ttyg',
+                        class: 'select-ttyg-agent-guide-dialog',
+                        skipPoint: true
+                    }, options)
+                },
+                {
+                    guideBlockName: 'clickable-element',
+                    options: angular.extend({}, {
+                        content: 'guide.step_plugin.select-ttyg-agent.open-agent-dropdown',
+                        class: 'open-agent-dropdown-guide-dialog',
+                        url: '/ttyg',
+                        elementSelector: GuideUtils.getGuideElementSelector('select-agent-dropdown')
```

Comment:
Add an empty onNextClick function to prevent the completion of the guide and wait for the user to click on the select agent dropdown.

Reply:
Change the code to be allow the user to return to step 1 if the selected agent is configured with missing repository.